src/scenes/lobbyscreen.ts

```diff
@@ -91,6 +91,7 @@ export class LobbyScreen extends Phaser.Scene {
     }
 
     private onStartButton(): void {
+        SessionManager.startGame();
```

Comment:
Instead i would here call a new method: PlayerManager.distributeRoles() and get rid of the startgame method.